### ISSUE TITLE
ngx-monaco-editor not working with angular 6.1

### ISSUE THREAD

**Issue by lesv** [atularen/ngx-monaco-editor#79]
I've downloaded the sample you posted in #56 and it works.  I've been using that to update my try and am finding the major difference is 6.1. (I'll update the issue once I gather more info.)

Looking at [this](https://update.angular.io/) suggests:

- [ ] Switch from HttpModule and the Http service to HttpClientModule and the HttpClient service. HttpClient simplifies the default ergonomics (You don't need to map to json anymore) and now supports typed return values and interceptors. Read more on angular.io

- [ ] Remove deprecated RxJS 6 features using rxjs-tslint auto update rules

For most applications this will mean running the following two commands:
```
npm install -g rxjs-tslint
rxjs-5-to-6-migrate -p src/tsconfig.app.json
```

- [ ] Once you and all of your dependencies have updated to RxJS 6, remove rxjs-compat.

### During the Update
There aren't currently any recommendations for moving between these versions.

### After the Update

- [ ] If you use the Angular Service worker, migrate any versionedFiles to the files array. The behavior is the same.

**Comment by lesv** [comment]
Found #73 where @dannymcgee has created a fork that updates things.

I'm not seeing any updates to this since May, wondering if it's abandoned?

**Comment by lesv** [comment]
You don't have issues open on your repo.

I've tried doing: `npm install github.com:dannymcgee/ngx-monaco-editor --save` -- No joy, so I'm trying to build it.

**Comment by dannymcgee** [comment]
Whoops, issues are open now. 

Yeah, I didn't publish to NPM or anything so you'll need to clone the repo and build it. I should probably edit the Readme to reflect that. Honestly, I've already switched over to CodeMirror for my particular project because I found it suits my needs better, otherwise I might consider publishing it as a package under a new name.

**Comment by dannymcgee** [comment]
@lesv Actually, now that I think of it, it will probably be a lot less headache to just incorporate the source files directly into your project as a regular Angular module. You can check the files under src/app for an example of how to do that.

**Comment by lesv** [comment]
Will look at that or using Codemirror (thanks for the pointer).  Which project do you use?  https://github.com/TypeCtrl/ngx-codemirror

**Comment by dannymcgee** [comment]
Yep, that's the one. Monaco is definitely a more powerful editor, but I found that CodeMirror's syntax highlighting was more to my liking and much more flexible to modify to exactly my needs than Monaco's Monarch system. Since I'm using it primary to **display** code rather than write it, the syntax highlighting was more important to me than Monaco's Intellisense, etc.
